corehq/apps/sms/models.py

```diff
@@ -1913,9 +1913,10 @@ def load_default_by_phone_and_domain(cls, backend_type, phone_number, domain=Non
             backend = cls.load_default_backend(backend_type, phone_number)
 
         if not backend:
-            raise BadSMSConfigException("No suitable backend found for phone "
-                                        "number and domain %s, %s" %
-                                        (phone_number, domain))
+            raise BadSMSConfigException("No gateway found "
+                                        "for phone number %s and domain %s. "
+                                        "To configure a gateway, please visit the SMS Connectivity Settings."
+                                        % (phone_number, domain))
```

Comment:
I wonder if this message should be translated, although I'm not sure that's simple when wrapped in an exception like this?